docs/code-signing.rst

```diff
@@ -0,0 +1,10 @@
+Code signing policy
+===================
+
+The code used by the DebOps Project and commited in its repositories is
+expected to be signed by a valid GPG key of a DebOps Developer or DebOps
+Contributor. This should allow for secure code authentication and a valid audit
```

Comment:
For key expiry, GPG keys can be stored in a "retired" keyring in the repository, similar to how [Debian does it](https://anonscm.debian.org/git/keyring/keyring.git/tree/). This could be used to authenticate commits signed by past keys.

Code signing for me is in part for a sane audit trail, so we can tell for example who merged a security hole, for example.

Other thing I'm thinking about is that since project is highly distributed (repositories on GitHub won't be there forever), moving the "trust" to GPG signatures stored with the `gitt` repository could allow for more secure distributed development. Some thoughts on code signing can be found in [A Git Horror Story](https://mikegerwitz.com/papers/git-horror-story) blog post. Since it's @ypid that convinced me to start signing my code, I imagine he could add his own thoughts here as well.